gh001-workflows/caller-workflows/release.yaml

```diff
@@ -0,0 +1,26 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+#    paths:
+#      - "*.py"
+      
+
+permissions: read-all
+
+jobs:
+  release:
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
```

Comment:
Where is the docker build and push?

Reply:
Docker build, push and upload as release asset steps are here: https://github.com/boldlink/workflows-and-pipelines/pull/1/files#diff-fa6cfde7188c7e07285ff255e260580eff6da670fffd0032f1402d93ca4acc90R67-R98